app/models/relationship.rb

```diff
@@ -326,6 +327,19 @@ def move_to_same_project_as_main
     end
   end
 
+  def move_explainers_to_source
```

Comment:
@melsawy it's better to add a condition here to do this operation only in two cases:

1. A new relationship is being created
2. A relationship is being updated from suggestion to confirmed

Reply:
@caiosba there is another case for `pin item`  which means sawp `source_id` & `target_id` so I only check the relationship type to be confirmed.
Is this Ok or I should handle these three cases?

Reply:
I think it's better to check the 3 cases for clarity and efficiency, Sawy.

Reply:
Done